engine/CMakeLists.txt

```diff
@@ -3,26 +3,35 @@
 
 option(BUILD_ENGINE_SAMPLES "Build cubos engine samples" OFF)
 
-message( "# Building engine samples: " ${BUILD_ENGINE_SAMPLES})
+message("# Building engine samples: " ${BUILD_ENGINE_SAMPLES})
 
 # Set engine source files
 
 set(CUBOS_ENGINE_SOURCE
     "src/cubos/engine/cubos.cpp"
-)
+
+    src/cubos/engine/rendering/renderer.cpp
```

Comment:
Change `rendering` to `gl` in order to have shorter names and to be consistent with `core` namespace naming